packages/nextjs/src/adapter-nextjs/page-props/galleryAlbum.ts

```diff
@@ -87,3 +88,7 @@ export async function getGalleryAlbumPageStaticPaths(): Promise<GetStaticPathsRe
         fallback: 'blocking',
     };
 }
+
+function isGalleryEmpty(gallery: NewsroomGallery) {
+    return gallery.images_number === 0 && gallery.videos_number === 0;
+}
```

Comment:
I believe this one should be extracted to utils and exported from the core library, like `getGalleryThumbnail`